customersupport/controllers/ticket.py

```diff
@@ -48,3 +84,15 @@ def get_ticket():
         res = Ticket.query.all()
     return jsonify([r.serialize() for r in res])
 
+@app.route('/api/refund/report', methods=["POST"])
```

Comment:
Maybe I'm missing it, but is this every being used? I think we are calling the HR wrapper function directly from the API above. I don't see why this should be exposed as an API.